resources/assets/js/back/app.js

```diff
@@ -15,27 +15,23 @@ import { query } from 'spatie-dom';
 if (query('blender-media')) {
     require.ensure([], () => {
         require('./modules/media').default();
-    }, 'back.media');
+    });
```

Comment:
Could be usefull to quickly 
- check filesize for chunks in cli output
- allocate browser errors to a certain chunk